examples/python/pre-filter/pre-filter/handler.py

```diff
@@ -150,12 +150,17 @@ def filterVcObject(obj, filter):
         moType = getManagedObjectTypeName(obj)
         objPath = getViObjectPath(obj)
         debug(f'{bgc.OKBLUE}{moType} Path > {bgc.ENDC}{objPath}')
-        if not re.search(filter, objPath):
-            debug(f'{bgc.WARNING}Filter "{filter}" does not match {moType} path "{objPath}". Exiting{bgc.ENDC}')
-            return f'Filter "{filter}" does not match {moType} path "{objPath}"', 200
-        else:
-            debug(f'{bgc.OKBLUE}Match > {bgc.ENDC}Filter matched {moType} path')
-            return True
+        try:
+            if not re.search(filter, objPath):
+                debug(f'{bgc.WARNING}Filter "{filter}" does not match {moType} path "{objPath}". Exiting{bgc.ENDC}')
+                return f'Filter "{filter}" does not match {moType} path "{objPath}"', 200
+            else:
+                debug(f'{bgc.OKBLUE}Match > {bgc.ENDC}Filter matched {moType} path')
+                return True
```

Comment:
Do you really want to return `True` here which seems to be used to return from the handler at a later point?

Reply:
I don't think it matters really - I only test if the result is a tuple later in line 216, any other result is ignored. That said this return is superfluous so I'll delete it!

I'll commit that one change a squash shortly. Many thanks!